clients/validator/examples/set_contract_upload_permissions.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import json
+
+# This script alters the genesis file so that only user account "dave"
+# can upload smart contracts.
+
+genesis_filename = "/home/dave/.nymd/config/genesis.json"
```

Comment:
As I mentioned before, I think this should rather use current user's home directory